qcodes/instrument_drivers/oxford/triton.py

```diff
@@ -163,30 +164,43 @@ def _get_named_channels(self):
                                    get_parser=self._parse_temp)
 
     def _get_pressure_channels(self):
+        self.chan_pressure = []
         for i in range(1, 7):
             chan = 'P%d' % i
+            self.chan_pressure.append(chan)
             self.add_parameter(name=chan,
                                unit='bar',
                                get_cmd='READ:DEV:%s:PRES:SIG:PRES' % chan,
                                get_parser=self._parse_pres)
+        self.chan_pressure = set(self.chan_pressure)
 
-    def _get_temp_channels(self, file):
+    def _get_temp_channel_names(self, file):
         config = configparser.ConfigParser()
-        with open(file, 'r') as f:
+        with open(file, 'r', encoding='utf16') as f:
             next(f)
             config.read_file(f)
 
         for section in config.sections():
             options = config.options(section)
             namestr = '"m_lpszname"'
             if namestr in options:
-                chan = 'T'+section.split('\\')[-1].split('[')[-1]
+                chan_number = int(section.split('\\')[-1].split('[')[-1]) + 1
+                # the names used in the register file are base 0 but the api and the gui
+                # uses base one names so add one
+                chan = 'T'+ str(chan_number)
                 name = config.get(section, '"m_lpszname"').strip("\"")
-                self.chan_temps[chan] = {'name': name, 'value': None}
-                self.add_parameter(name=chan,
-                                   unit='K',
-                                   get_cmd='READ:DEV:%s:TEMP:SIG:TEMP' % chan,
-                                   get_parser=self._parse_temp)
+                self.chan_temp_names[chan] = {'name': name, 'value': None}
+
+    def _get_temp_channels(self):
+        self.chan_temps = []
+        for i in range(1, 16):
```

Comment:
This should probably be range(1,17) so that all 16 channels are included. We are not using T16 at Triton 10 but anyway